src/io/flutter/module/InstallSdkAction.java

```diff
@@ -143,6 +145,7 @@ public void validateSelectedFiles(VirtualFile[] files) throws Exception {
 
       final VirtualFile installTarget = FileChooser.chooseFile(descriptor, null, null);
       if (installTarget != null) {
+        FlutterInitializer.sendAnalyticsAction(getClass().getSimpleName());
```

Comment:
Exactly.  I *think* this is what we've got here. Separate calls for:

1. ViewDocsAction, and
1. GitCloneAction

But I may be misunderstanding.  Anyway, no need to try and review this on your phone.  Let's look at it tomorrow. 👍